File: app/src/main/java/com/kimjisub/launchpad/midi/driver/Noting.java
package com.kimjisub.launchpad.midi.driver;

public class Noting extends DriverRef {
	@Override
	public void getSignal(int cmd, int sig, int note, int velo) {
	}

	@Override
	public void sendPadLED(int x, int y, int velo) {
	}

	@Override
	public void sendChainLED(int c, int velo) {
	}

	@Override
	public void sendFunctionkeyLED(int f, int velo) {
	}

	@Override
	public void sendClearLED() {
	}

}

File: app/src/main/java/com/kimjisub/launchpad/midi/driver/Noting.kt
package com.kimjisub.launchpad.midi.driver

class Noting : DriverRef() {
	override fun getSignal(cmd: Int, sig: Int, note: Int, velocity: Int) {}
	override fun sendPadLed(x: Int, y: Int, velocity: Int) {}
	override fun sendChainLed(c: Int, velocity: Int) {}
	override fun sendFunctionkeyLed(f: Int, velocity: Int) {}
	override fun sendClearLed() {}
}